components/identities/availableIdentities.tsx

```diff
@@ -99,13 +97,24 @@ const AvailableIdentities = ({ tokenId }: { tokenId: string }) => {
     fetchProfilePic();
   }, [identity]);
 
+  // this function does not work
   const hideActionsHandler = (state: boolean) => {
```

Comment:
The function is empty? You can remove it, and thus remove it from the dependencies of other components